app/helpers/newsletter_helper.rb

```diff
@@ -0,0 +1,9 @@
+module NewsletterHelper
```

Comment:
Should we test this helper?

Reply:
> Should we test this helper?

Definitely. I'm not sure I understand the comment; the tests are already there, aren't they? :thinking:

By the way, we no longer write helper methods unless strictly necessary but use components instead. However, @gabrielcosta42, if you aren't familiar with components and don't feel like looking into them, you can leave the helper method and we'll move it to a component :ok_hand:.